### ISSUE TITLE
wrong popup appearing for Edit Tag feature

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/CWRC-Writer#434]
Embedded CWRC-Writer
Chrome on a Mac
Agnes Maule Machar writing doc: http://beta.cwrc.ca/islandora/object/islandora%3Af2c37376-d105-4baa-b338-e2f27ef74952/edit-cwrc#

Had clicked on "1899" in screenshot, which is a date entity (and got error to be logged separately). Then clicked within text of the title to the left of it, ctrl clicked, and got the popup for the date entity.

![image](https://cloud.githubusercontent.com/assets/545952/17939918/9d6bb130-69fa-11e6-9668-dd47f5eea31c.png)

**Comment by ajmacdonald** [comment]
So the issue is that it's showing you the entity editing dialogue and you want to see the standard tag editing dialogue?

**Comment by SusanBrown** [comment]
No, the issue is that my cursor is in the middle of a title tag (between Canadian and Poems) but the dialog that I have been given is the dialog for the date tag.
